### PR TITLE
Add script to convert activity-stream l10n into l10n-central

### PR DESCRIPTION
**Pull request by Pike**
This is for https://bugzilla.mozilla.org/show_bug.cgi?id=1457223,
and converts the data from https://github.com/mozilla/activity-stream-l10n/

This gets the party started.

### CODE REVIEW

File: mozilla_l10n/activity-stream-convert [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import subprocess
+import sys
+from tempfile import mkstemp
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--method', choices=['rebase', 'merge'], default='rebase')
+parser.add_argument('--push', action='store_true', help='push results upstream')
+parser.add_argument('-p', default=4, metavar='N', help='Run N parallel jobs')
+parser.add_argument('a_s', metavar='activity-stream-l10n')
+parser.add_argument('l10n', metavar='l10n-central')
```
Comment (flodolo) [inline mozilla_l10n/activity-stream-convert]: `as_path` and `l10n_path` for these two parameters would make things clearer

File: mozilla_l10n/activity-stream-convert [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+locales.sort()
+handle = []
+skip = []
+for loc in locales:
+    if os.path.isdir(os.path.join(args.l10n, loc)):
+        handle.append(loc)
+    else:
+        skip.append(loc)
+
+handle = handle[:4]
```
Comment (flodolo) [inline mozilla_l10n/activity-stream-convert]: Why this? I've never used ThreadPoolExecutor, but it seems like you're cutting down the list to 4 locales, not just taking 4 at a time.

**Comment by Pike** [comment]
> Should we just merge?

Really depends on your expectations on what lands in this repo and what not. The basic skeleton might come in helpful in the future, though.

**Comment by flodolo** [comment]
This repo is pretty much a collection of scripts, to use as-is or to build from. This fits right in :)

**Pull request merged**